features/support/shared_contexts/simple_git_repository.rb

```diff
@@ -36,7 +36,7 @@
   end
 
   def change(file_path)
-    file_path.write('changed')
+    file_path.write("'changed'")
```

Comment:
Why was this necessary?

Reply:
this file is loaded inside of `features/rspec_runner_spec.rb`
`changed` is not a valid ruby code :D
But `'changed'` is a valid string